### DISCUSSION TITLE
How to use Number Field and/or Date Picker for Post REST API bindings data?

### DISCUSSION THREAD

**Discussion by abdularis** [Budibase/budibase#8184] [Help]
Hi,

I'm new and just come across`budibase` it seems promising so I'm experimenting to integrate my REST API, but I have a hard time integrating POST API into form fields as it like always treating the query bindings as string, and I find it dificult to get any references about that, may be anyone has any experience with this.
to summarize:
Is it possible to integrate `Number Field' and` Date Picker` into bindings in POST REST Query?

Thank you

**Comment by melohagan** [comment]
Hey @abdularis 

I'm not entirely sure what you mean, but you can set the field types of your REST query schema.
![Screenshot 2022-10-10 at 15 54 21](https://user-images.githubusercontent.com/101575380/194894909-d63e8f29-1fa1-414f-8158-100a56f98a21.png)

After you post an example, click on the schema tab and select the data types for each field. Make sure to click **Save** when you're done!
